index.html

```diff
@@ -229,5 +229,40 @@ <h2 class="text-4xl mb-8 font-semibold">
         >
       </div>
     </div>
+    
+   <!-- new card in a container -->
+    
+    <div class=" container mx-auto w-auto px-5 py-24 flex flex-inline">
+        <div class="lg:w-1/2 bg-yellow-200 flex flex-inline">
+            <div class="lg:w-2/5 bg-green-200">
+                <h2>IMAGE</h2>
+            </div>
+            <div class="lg:w-3/4 px-5 pt-1 bg-green-500">
+                <h1 class="font-medium text-xl pt-8">awesome-github-profiles</h1>
+                <p class="font-light text-sm py-1">List of GitHub profiles that have awesome
+                    customisation, that you can use for inspiration.</p>
+                <ul class="lg:pb-5 lg:pt-2 ">
+                    <li class="py-1 flex flex-inline">
+                        <i class="material-icons">link</i>
+                        <a class="pl-2" href="">view repo</a>
+                    </li>
+                    <li class="py-1 flex flex-inline">
+                        <i class="material-icons">link</i>
```

Comment:
These icons needs to be replaced with FontAwesome icons

Reply:
yeah, i'll change it..

Reply:
👍

Reply:
i have changed it to fontaswesome icons

Reply:
ok great 👍